app/src/views/internships/index.vue

```diff
@@ -204,20 +207,27 @@ export default class extends Vue {
     this.getList();
   }
 
-  private handleModifyStatus(row: any, status: string) {
-    this.$message({
-      message: '操作成功',
+  private async handleDelete(row: any, status: string) {
+    await deleteInternship(row.id!);
+    const found = this.list.findIndex(s => s.id === row.id);
+    if (found !== -1)
+    {
+      this.list.splice(found, 1);
+    }
+    this.$notify({
+      title: 'Internship creation',
```

Comment:
Should be translate using i18n module + notify that a data is deleted, not created

``` javascript
title: this.$t('notify.internships.delete.title'),
message: this.$t('notify.internships.delete.msg'),
```